modules/ingester/instance_search_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/grafana/tempo/pkg/model/trace"
 	"github.com/grafana/tempo/pkg/tempopb"
 	v1 "github.com/grafana/tempo/pkg/tempopb/common/v1"
+	v11 "github.com/grafana/tempo/pkg/tempopb/trace/v1"
```

Comment:
`nitpick` we often use 'trace_v1', 'resource_v1` as import aliases
```suggestion
	trace_v1 "github.com/grafana/tempo/pkg/tempopb/trace/v1"
```

Reply:
ahhh I see, it's v11 in the pb.go files